### PR TITLE
Thuynh/franchise page: reset standings and fix historic standings

### PR DESCRIPTION
**Pull request by roundleague**
Thuynh/franchise page: reset standings and fix historic standings

### CODE REVIEW

File: pages/Teams/team-profile-page.cfm [hunk 1]
```diff
@@ -149,9 +185,83 @@
...
+                Where seasonID = #getTeamStandings.seasonID#
+              </cfquery>
+
+              <cfquery name="getPlayoffFinish" datasource="roundleague">
+                SELECT t.teamName, ps.seasonID, max(ps.bracketRoundID) as maxBracketRoundID, pb.MaxTeamSize, pb.Playoffs_bracketID
+                FROM playoffs_schedule ps
+                JOIN playoffs_bracket pb ON ps.Playoffs_BracketID = pb.Playoffs_bracketID
+                JOIN teams t ON t.teamId = <cfqueryparam cfsqltype="INTEGER" value="#url.teamID#">
+                WHERE (hometeamID = <cfqueryparam cfsqltype="INTEGER" value="#url.teamID#"> OR awayteamID = <cfqueryparam cfsqltype="INTEGER" value="#url.teamID#">)
+                AND ps.seasonID = #getTeamStandings.seasonID#
```
Comment (Jonan-Doan) [inline pages/Teams/team-profile-page.cfm]: Hey Tim, remember to use cfqueryparam (bad Tim)

**Review by Jonan-Doan — CHANGES_REQUESTED** [review]

File: pages/Teams/team-profile-page.cfm [hunk 2]
```diff
@@ -54,6 +54,41 @@
   GROUP BY lastName, firstName
 </cfquery>
 
+<cfquery name="getTeamDataM" datasource="roundleague">
```
Comment (Jwwason) [inline pages/Teams/team-profile-page.cfm]: "getTeamDataM" change to something that's easier to understand

File: pages/Teams/team-profile-page.js [hunk 3]
```diff
@@ -8,6 +8,10 @@ $( document ).ready(function() {
     openCity('PlayerStats')
   });
 
+  $(".franchiseInfoBtn").click(function(){
+    openCity('FranchiseInfo')
```
Comment (Jwwason) [inline pages/Teams/team-profile-page.js]: openCity should be renamed to openTab

File: pages/Teams/team-profile-page.js [hunk 4]
```diff
@@ -8,6 +8,10 @@ $( document ).ready(function() {
...
+  });
+
   function openCity(cityName) {
```
Comment (Jwwason) [inline pages/Teams/team-profile-page.js]: openCity should be renamed to openTab

**Review by Jwwason — CHANGES_REQUESTED** [review]
Nice, please fix some minor mistakes

File: pages/Teams/team-profile-page.cfm (hunk 2 above)
Comment (roundleague) [reply pages/Teams/team-profile-page.cfm]: thanks for the catch dude!

**Pull request merged**